### PR TITLE
[reopen] make [envOrElse, propOrElse] call-by-name

### PR DESCRIPTION
**Pull request by 193s**
:octopus: [reopen] #4278

### CODE REVIEW

File: src/library/scala/util/Properties.scala [hunk 1]
```diff
@@ -57,7 +57,7 @@ private[scala] trait PropertiesTrait {
   def setProp(name: String, value: String)      = System.setProperty(name, value)
   def clearProp(name: String)                   = System.clearProperty(name)
 
-  def envOrElse(name: String, alt: String)      = Option(System getenv name) getOrElse alt
+  def envOrElse(name: String, alt: => String)   = Option(System getenv name) getOrElse alt
   def envOrNone(name: String)                   = Option(System getenv name)
 
   def envOrSome(name: String, alt: Option[String])       = envOrNone(name) orElse alt
```
Comment (som-snytt) [inline src/library/scala/util/Properties.scala:63]: You missed one?

**Comment by adriaanm** [comment]
Hi! PR validation requires #4272 be merged first (will do so on Monday). Unfortunately, you'll have to rebase once it is.

**Comment by retronym** [comment]
#4272 has now been merged. Can you please rebase on the latest 2.12.x branch now? Sorry for the hassle.

**Comment by 193s** [comment]
Sorry, my mistake... I'll reopen soon..

**Comment by 193s** [comment]
I made branch topic/prop2 based 2.12.x and opened #4285
I'm very sorry :sob:

**Comment by retronym** [comment]
No apology necessary, thanks for the contribution!

**Pull request closed without merging**